### ISSUE TITLE
Build Failure: Multiple dex files define Lcom/example/speechassist/BuildConfig

### ISSUE THREAD

**Issue by nasradev** [gsssrao/UnityAndroidSpeechRecognition#2]
Hey, many thanks for open sourcing this! I tried creating a new project in Unity 2018, imported your plugin, set my bundle name, set min sdk to 21 and target sdk to 23, and pressed build. Unfortunately I get this error:

Build failure: com.android.build.api.transform.TransformException: com.android.ide.common.process.ProcessException: java.util.concurrent.ExecutionException: com.android.dex.DexException: Multiple dex files define Lcom/example/speechassist/BuildConfig; See the Console for details.

Any idea what this could be due to? Many thanks

**Comment by gsssrao** [comment]
It seems like there are some conflicting files.

Could you remind me about this again on Wednesday (I am a bit busy till then)? 

I had tested this plugin on Unity 5.1.1.f1 at the time of creating this repository. I can try to test it on Unity 2018 and let you know.

**Comment by nasradev** [comment]
Hey, thanks for looking into this. Reminding as requested :).

**Comment by gsssrao** [comment]
Hey @nasradev, thanks for the reminder. 

I was able to replicate your issue. As mentioned [here](https://forum.unity.com/threads/android-project-multiple-dex-files-define-lcom-android-vending-billing-buildconfig.408823/), deleting the BuildConfig.class from within the .jar solved it. In the process, I also cleaned the code and added some new features. I have tested the latest version on `Unity 2018.1.0f2` and it works. 

Let me know if it works on your end or if you face any problems.
